packages/manager/apps/dedicated/client/app/dedicated/server/terminate/dedicated-server-terminate.html

```diff
@@ -7,15 +7,24 @@
     >
         <div
             data-wizard-step
-            data-wizard-step-valid="ctrl.cancelSubscriptionForm.cancelMethod"
+            data-wizard-step-valid="ctrl.cancelSubscriptionForm.cancelMethod && !ctrl.terminationUnavailable && !ctrl.loading"
         >
             <div
                 class="text-center"
-                data-ng-if="ctrl.cancelSubscriptionForm.isSubmiting"
+                data-ng-if="ctrl.cancelSubscriptionForm.isSubmiting || ctrl.loading"
             >
                 <oui-spinner></oui-spinner>
             </div>
 
+            <oui-message
+                class="mb-3"
+                data-ng-if="ctrl.terminationUnavailable"
+                data-type="error"
+            >
+                <span data-translate="server_close_service_enterprise"></span>
+                <span>(Account Manager).</span>
```

Comment:
This trad is directly in DOM and not in a trad file, is this expected ?

Reply:
Yes... Since this is going to remain the same for all languages, it was decided that we will have it in the DOM directly